### PR TITLE
Fix export to use `list_query` option

### PR DESCRIPTION
**Pull request by villqrd**
This PR addresses the issue described in https://github.com/aminalaee/sqladmin/issues/380

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -73,6 +73,7 @@ dependencies = [
   "mypy==0.982",
   "psycopg2==2.9.5",
   "pytest==7.2.0",
+  "pytest-asyncio==0.20.2",
```
Comment (aminalaee) [inline pyproject.toml:76]: I don't think this is needed, there's pytest anyio already used in other places. The same can be used here.

File: tests/test_models.py [hunk 2]
```diff
@@ -57,6 +57,13 @@ def prepare_database() -> Generator[None, None, None]:
     Base.metadata.drop_all(engine)
 
 
+@pytest.fixture()
+def session(prepare_database) -> LocalSession:
```
Comment (aminalaee) [inline tests/test_models.py:61]: Does it have to be a fixture? Can't it be instantiated within the test?

File: tests/test_models.py [hunk 3]
```diff
@@ -483,3 +490,30 @@ class UserAdmin(ModelView, model=User):
...
+    batman = User(name="batman")
+    bruce = User(name="bruce wayne")
+    superman = User(name="superman")
+    session.add(batman)
+    session.add(bruce)
+    session.add(superman)
+    session.commit()
+    session.refresh(batman)
+    session.refresh(bruce)
+    session.refresh(superman)
```
Comment (aminalaee) [inline tests/test_models.py:506]: It can simply work with one instance to prove the test. No need for the extra LOC.

**Review by aminalaee — CHANGES_REQUESTED** [review]
Thanks a lot for the contribution

File: pyproject.toml (hunk 1 above)
Comment (villqrd) [reply pyproject.toml:76]: indeed, i was wondering why your async tests were working and not mine, thanks for pointing out the secret! will change

File: tests/test_models.py (hunk 3 above)
Comment (villqrd) [reply tests/test_models.py]: see last changes

**Review by aminalaee — APPROVED** [review]
Thanks a lot for the contribution 🎉

**Pull request merged**